docs/sdk/core-kit/mpc-core-kit/usage.mdx

```diff
@@ -0,0 +1,294 @@
+sidebar_label: "Usage"
+displayed_sidebar: sdk
+description: "Web3Auth MPC Core Kit Web SDK - Usage | Documentation - Web3Auth"
+---
+
+import LogOut from "@site/src/common/sdk/pnp/web/_logout.mdx";
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+Once you've installed and successfully initialized `Web3AuthMPCCoreKit`, you can use it to authenticate your users. Further, you can use the native
+provider given by Web3Auth to sign transactions and interact with the blockchain.
+
+Natively, the instance of `Web3AuthMPCCoreKit` (referred to as `coreKitInstance` in our examples) returns the following functions:
+
+Authentication Functions:
+
+- `connect()` - Logs in the User.
+- `logout()` - Logs out the User.
+- `handleRedirectResult()` - Handles the redirect result while logging in the User, while in redirect mode
+
+Backup Share Functions:
+
+- `exportBackupShare()` - Exports the Backup Mnemonic Share.
+- `inputBackupShare()` - Inputs the Backup Mnemonic Share to recover the User's account.
+
+Security Question Functions:
+
+- `addSecurityQuestionShare()` - Adds a Security Question Share to the User's account.
+- `recoverSecurityQuestionShare()` - Recovers the User's account using the Security Question Share.
+- `changeSecurityQuestionShare()` - Changes the Security Question Share of the User's account.
+- `deleteSecurityQuestionShare()` - Deletes the Security Question Share of the User's account.
+
+User Functions:
+
+- `getUserInfo()` - Returns the User's information received from the ID Token.
+- `getKeyDetails()` - Returns the User's Key Details.
+- `commitChanges()` - Commits the changes made to the User's account while in manual sync mode.
+- `CRITICAL_resetAccount()` - Resets the User's account. This function is only to be used in case of emergencies, when the user is unable to recover
+  their account. This deletes all the history of the previous key and the funds/ assets associated with it.
+
+## Logging in your User
+
+#### `connect(params: LoginParams): Promise<SafeEventEmitterProvider | null>;`
+
+To login a user in the Web3Auth MPC Core Kit SDK, you need to call `connect()` function. The function takes in the following parameters:
+
+| Variable      | Type     | Description      | Mandatory |
+| ------------- | -------- | ---------------- | --------- |
+| `loginParams` | `object` | Login Parameters | Yes       |
+
+:::tip
+
+Checkout the [Authentication Section](./authentication) to know more about the `loginParams` object.
+
+:::
+
+### Returns
+
+```js
+connect(loginParams: LoginParams): Promise<SafeEventEmitterProvider | null>
+```
+
+On successful login, the `connect()` function returns a `SafeEventEmitterProvider` instance. This instance contains the respective provider
+corresponding to your selected blockchain. You can use this provider to connect your user to the blockchain and make transactions.
+
+On unsuccessful login, this function will return a `null` value.
+
+### Example
```

Comment:
These examples can go one level deep to avoid clutter on the right side, which can be distracting.

<img width="409" alt="Screenshot 2023-05-18 at 2 07 26 PM" src="https://github.com/Web3Auth/web3auth-docs/assets/6962565/7a8e4b8a-a57d-41ab-b102-e0fdf2125201">

Reply:
makes sense, fixed